careerfairsystems/arkad-app
# Listview company frontend

## Change description

> A user can now login and logout and will get a list view when loged in

## How to verify

> Try to login and logout and see if the listview appears. Also try wrong password.

## Issues fixed

> #166
